pynecone/app.py

```diff
@@ -316,6 +321,34 @@ def get_load_events(self, route: str) -> List[EventHandler]:
             route = constants.INDEX_ROUTE
         return self.load_events.get(route, [])
 
+    def add_download_file(self, download_url: str, file_path: str):
+        """Download file from host by downloa_durl.
+
+        Args:
+            download_url: The route to get the file.
+            file_path: The file from host.
+
+        Returns:
+            Respone a file from server.
+
+        Raises:
+            ValueError: download_url already is used.
+        """
+        if download_url in self.backend_router:
+            raise ValueError(f"{download_url} already is used.")
+
+        self.backend_router.add(download_url)
+
+        @self.api.get(f"{download_url}")
+        async def response():
+            return FileResponse(
+                path=file_path,
+                filename=os.path.split(file_path)[1],
```

Comment:
why `[1]`? Seems like you would want `[-1]` to grab the right-most element of the list. At any rate, I'd prefer to see `pathlib.Path(file_path).name` here instead.

Reply:
I agree with you. Thanks for your suggestion.